draft-irtf-cfrg-frost.md

```diff
@@ -552,7 +552,8 @@ procedure to produce its own signature share.
 
 The output of this procedure is a signature share and group commitment share.
 Each signer then sends these shares back to the collector; see
-{{encode-sig-share}} for encoding recommendations.
+{{encode-sig-share}} for encoding recommendations. Each signer MUST delete
+the nonce and corresponding commitment after this round completes.
```

Comment:
```suggestion
the nonce and SHOULD delete the corresponding commitment after this round completes.
```

Reply:
Clarify to ensure that nonces are used only once.

Reply:
Done in #98.